### DISCUSSION TITLE
Location of Nodes in network?

### DISCUSSION THREAD

**Discussion by Carlmcap** [filecoin-project/community#285] [Developers]
Hey Everyone,

Im hoping to see if I can locate the country of the nodes used in the network? 

More specifically, if you are able to identify the country of specific nodes.

I have been through the docs, and could not identify anything specifically. Let me know if you can point me in the right direction.

**Comment by rjan90** [comment]
Hey @Carlmcap! You can check out Storage Providers on a global map on here: 
- https://observablehq.com/@jimpick/miner-report-miners-on-a-global-map?collection=@jimpick/miner-report
- https://m.filscout.com/en/peer/map

For people that are just running the Lotus-node, I’m not certain if there is a map yet

**Reply by Carlmcap** [reply]
@rjan90 thanks ! Im going to reach out to the individual in the article and see if there is a way to show me how to pull the raw data.

Fascinating stuff here!
